tests/bud.bats

```diff
@@ -366,6 +366,26 @@ _EOF
   expect_output --substring "invalid response status"
 }
 
+@test "build test has gid in supplemental groups" {
+  _prefetch alpine
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+  run_buildah build $WITH_POLICY_JSON -t source -f $BUDFILES/supplemental-groups/Dockerfile
+  # gid 1000 must be in supplemental groups
+  expect_output --substring "Groups:	1000"
+}
+
+@test "build test has gid in supplemental groups and --isolation chroot" {
```

Comment:
Non-blocker: try to avoid giving tests names that are exact substrings of the names of other tests, which makes using `bats -f` to run specific tests harder.

Reply:
Changed the test name from first one.